Builds/CMake/RippledCore.cmake

```diff
@@ -844,6 +844,7 @@ target_sources (rippled PRIVATE
   src/test/nodestore/Backend_test.cpp
   src/test/nodestore/Basics_test.cpp
   src/test/nodestore/Database_test.cpp
+  src/test/nodestore/DatabaseShard_test.cpp
```

Comment:
Nit: this should go before Database_test.cpp

Reply:
Done